Source/WebCore/rendering/GridTrackSizingAlgorithm.cpp

```diff
@@ -1355,6 +1356,37 @@ static inline double normalizedFlexFraction(const GridTrack& track)
     return track.baseSize() / std::max<double>(1, flexFactor);
 }
 
+// FIXME: https://bugs.webkit.org/show_bug.cgi?id=282395
+void IndefiniteSizeStrategy::accumulateFlexFractionMasonry(double& flexFraction, unsigned columnIndex, GridTrackSizingDirection direction, GridLayoutState& gridLayoutState) const
+{
+    auto trackLength = m_algorithm.tracks(direction).size();
```

Comment:
To be more consistent with the rest of the grid code: 
```suggestion
    auto numTracks = m_algorithm.tracks(direction).size();
```